aioshelly/rpc_device/device.py

```diff
@@ -233,6 +241,10 @@ async def update_config(self) -> None:
         """Get device config from 'Shelly.GetConfig'."""
         self._config = await self.call_rpc("Shelly.GetConfig")
 
+    async def update_info(self) -> None:
+        """Get device status from 'Shelly.GetDeviceInfo'."""
+        self._info = await self.call_rpc("Shelly.GetDeviceInfo")
+
```

Comment:
The output of this endpoint is identical to `/shelly` and we already fetch at  https://github.com/home-assistant-libs/aioshelly/blob/1f0b69a1541b32b5c9a03752f46f1b9fc9573883/aioshelly/rpc_device/device.py#L155
We used `/shelly` since older devices didn't expose the `Shelly.GetDeviceInfo` (or at least without authentication and this allows us to have an identical path for gen1/2/3

Reply:
Unfortunately ".sys.app" is only present on the `DeviceInfo`. Tested on 5 different models :-(

Reply:
BTW, we need the "app" model only for gen2, gen1 provides a single URL for all devices.

Reply:
I checked few devices and they all have the same output which also includes the `app` would be better to first ask Shelly why you see a difference on your devices

Reply:
Let me recheck,if it's /shelly is a lot better

Reply:
Double checked on `WallDisplay` and on `BLU Gateway`, both missing.

Reply:
WallDisplay is an Android device, it impelents the next-gen apis but is based on a different platform and does not shrare code. `app` may not be there.

BLU Gateway does have the `app` property in `/shelly`

Reply:
Wall Display was on a old firmware, latest works fine.

BLU Gateway needed a hard reset, now works fine.